x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/AutoConfigureNode.java

```diff
@@ -882,9 +882,9 @@ protected static boolean anyRemoteHostNodeAddress(List<String> allNodesTransport
 
     protected String hostSettingValue(InetAddress[] allAddresses) {
         if (Arrays.stream(allAddresses).anyMatch(InetAddress::isSiteLocalAddress)) {
```

Comment:
Is the `if` condition required?
I think filling in `0.0.0.0` all the time is OK?

Reply:
That's true @albertzaharovits , thanks!